x-pack/plugins/code/public/monaco/monaco.ts

```diff
@@ -114,46 +119,72 @@ import 'monaco-editor/esm/vs/basic-languages/typescript/typescript.contribution'
   },
 };*/
 
-const elasticLight = {
-  keyword: '#DD0A73',
-  comment: '#90A4AE',
-  delimeter: '#017F75',
-  string: '#0079A5',
-  number: '#E5830E',
-  regexp: '#0079A5',
-  types: '#909AA1',
-  annotation: '#D9D9D9',
-  tag: '#DD0A73',
-  symbol: '#A30000',
-  foreground: '#3F3F3F',
-  editorBackground: '#FFFFFF',
-  lineNumbers: '#D9D9D9',
+function getTheme() {
+  if (IS_DARK_THEME) {
+    return darkTheme;
+  } else {
+    return lightTheme;
+  }
+}
+
+function getHex(rgbString: string) {
+  const colorValues = rgbString.slice(4, -1);
+  const colorArray = colorValues.split(', ');
+
+  // @ts-ignore
+  return '#' + convert.rgb.hex(Number(colorArray[0]), Number(colorArray[1]), Number(colorArray[2]));
+}
+
```

Comment:
it probably worth it to add a const

const themeName = getTheme()

then, you could use themeName instead of getTheme() to reduce extra computation

Reply:
I like that. I'll get that in there.